collector/nodes.go

```diff
@@ -1038,6 +1128,22 @@ func (c *Nodes) Collect(ch chan<- prometheus.Metric) {
 				metric.Labels(nodeStatsResponse.ClusterName, node)...,
 			)
 		}
+		for _, metric := range c.nodeMetricsVec {
+			labelValues := metric.LabelValues(nodeStatsResponse.ClusterName, node)
+			values := metric.Values(node)
+
+			for i, labelValue := range labelValues {
+				counterVec := prometheus.NewCounterVec(prometheus.CounterOpts{
+					Namespace: namespace,
+					Subsystem: metric.Subsystem,
+					Name:      metric.Name,
+					Help:      metric.Help},
+					metric.Labels)
+				counterVec.WithLabelValues(labelValue...).Set(values[i])
+				counterVec.Collect(ch)
+			}
+
```

Comment:
this metric returns false results regarding the labels: on our cluster we get: 
```
# TYPE elasticsearch_indices_query_cache_count counter
elasticsearch_indices_query_cache_count{category="hit",cluster="cluster",host="host",name="name"} 3.1548049e+07
elasticsearch_indices_query_cache_count{category="miss",cluster="cluster",host="host",name="name"} 4.9529376e+07
```
(the label values for host and name are not (!) redacted)